src/ContentPage/index.js

```diff
@@ -25,6 +25,21 @@ const style = css`
   div.BookingInfo {
     width: 548px;
   }
+
+  @media only screen and (min-device-width: 320px) and (max-device-width: 480px) {
```

Comment:
what about using `min-width` and `max-width`?

Reply:
You mean `min-width` instead of `min-device-width` in every media query (same for `max-.*`)?

Reply:
I used `min-device-width` because that's what's used all over the project, however I think this is a very good idea and I can't think of any reason why we should use `-device-`, so I replaced it project-wide.

I think this pull request should be squashed, but maybe I wouldn't squash the project-wide replacement into the same one commit. What do you think? Maybe I should just move that commit to another pull request.

Reply:
IMHO it's fine to squash